src/cryptonote_core/cryptonote_core.cpp

```diff
@@ -180,6 +180,14 @@ namespace cryptonote
     "service-node"
   , "Run as a service node"
   };
+  static const command_line::arg_descriptor<std::string> arg_public_ip = {
+    "public-ip"
+  , "Globally reachable IP address for incoming storage server requests (required for Service Nodes)"
+  };
+  static const command_line::arg_descriptor<uint16_t> arg_sn_bind_port = {
+    "sn-bind-port"
+  , "Port a storage server instance is listening on. Note that it is a Service Node's responsibility to make sure it is properly configured (Loki Launcher does that automatically)."
```

Comment:
This argument seems misnamed: I think it should be `storage-server-port` (it is neither the service node's port, nor is it a bind port).

Also the help description here could use some improvement; how about:

"The port that this service node's storage server is accessible on.  A listening storage server is required for service nodes.  (This option is specified automatically when using Loki Launcher)."

Reply:
Updated.